thz/__init__.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python3
+#########################################################################
+#  Copyright 2014 toggle
+#########################################################################
+#
+#  THZ plugin for SmartHome.py.   http://mknx.github.com/smarthome/
+#
+#  This plugin is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This plugin is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this plugin. If not, see <http://www.gnu.org/licenses/>.
+#########################################################################
+
+import logging
+import time
+import sys
+import datetime
+from . import ThzProtocol
+import socket as socket
+from select import select
+import threading
+from lib.model.smartplugin import SmartPlugin
+
+logger = logging.getLogger('THZ')
```

Comment:
Please move the logger into the `SmartPlugin` class `THZ`.

Reply:
Inside the plugin class the logger should be initialized this way:
```
        self.logger = logging.getLogger(__name__)
```

This way the logger is named `plugins.thz`. This standardization is helpful for an upcoming graphical logging configuration.